secgroups.go

```diff
@@ -218,6 +218,23 @@ func (c *Client) BindSecGroup(secGUID, spaceGUID string) error {
 	return nil
 }
 
+/*
+BindSpaceStagingSecGroup contacts the CF endpoint to associate a space with a security group for staging functions only
+secGUID: identifies the security group to add a space to
+spaceGUID: identifies the space to associate
+*/
+func (c *Client) BindSpaceStagingSecGroup(secGUID, spaceGUID string) error {
```

Comment:
BindStagingSecGroupToSpace?

Reply:
I like that better, will make the changes -- also am going to add changes to the model to load them (like we currently do with spaces)